binding.gyp

```diff
@@ -2,18 +2,20 @@
   "targets": [
     {
       "target_name": "tree_sitter_comment_binding",
+      "dependencies": [
+        "<!(node -p \"require('node-addon-api').targets\"):node_addon_api_except",
+      ],
       "include_dirs": [
-        "<!(node -e \"require('nan')\")",
-        "src"
+        "src",
       ],
       "sources": [
-        "src/parser.c",
         "bindings/node/binding.cc",
-        "src/scanner.c"
```

Comment:
you have to re-add the scanner manually here; `ts g` unfortunately overwrites that section.

Reply:
thank you!